web/templates/package/show.html.eex

```diff
@@ -6,7 +6,12 @@ licenses = @package.meta["licenses"] || []
 links = Enum.to_list(@package.meta["links"] || [])
 build_tools = @current_release && @current_release.meta["build_tools"] || []
 
-tools = [mix: "mix.exs", rebar: "rebar.config", erlang_mk: "erlang.mk"]
+tools = if Enum.any?(build_tools, fn(tool) -> Enum.member?(["rebar", "rebar3", "make"], tool) end) do
+           [mix: "mix.exs", rebar: "rebar.config", erlang_mk: "erlang.mk"]
+        else
+           [mix: "mix.exs", erlang_mk: "erlang.mk"]
+        end
+
```

Comment:
Rebar can't build make projects and we shouldn't list erlang.mk when it's a mix project. Can you also move the condition to its own line, it's a bit hard to read as it is now.